src/services/Libs.ts

```diff
@@ -176,6 +184,41 @@ export const getHostname = (urlToGetHostName: string | undefined): string => {
   }
 };
 
+/**
+ * Gets the default expression options depending on the list/storeId.
+ * If storeId is not default, it will try to get defaults set in default list
+ * before using CAD defaults (all checked).
+ * @param state The State (store.getState())
+ * @param storeId The container id, or 'default'
+ * @param listType The List Type
+ */
+export const getContainerExpressionDefault = (
+  state: State,
+  storeId: string,
+  listType: ListType,
+): Expression => {
+  const getExpression = (list: string): Expression | undefined => {
```

Comment:
I haven't done TS in a while, but I think that you can do `Expression?` rather than `Expression | undefined`. If this is wrong, feel free to ignore this comment.

Reply:
What you're referencing is a function return type, so that doesn't work apparently.